.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -10,6 +10,12 @@ Tips:
 - Please include details about the environment you're running in.
 -->
 
+<!--
+Note:
+
+If you are an AWS customer and experiencing an urgent issue you believe is related to AWS, please [contact support](https://aws.amazon.com/contact-us/) in addition to filing this issue.
```

Comment:
Realized this isn't going to render as a hyperlink inside the comment. Slight tweak coming.